# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../k3b/flac-vorbiscomment.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- k3b-0.11.21/src/audiodecoding/flac/k3bflacdecoder.cpp.orig	2005-03-12 17:52:22.000000000 +0100
+++ k3b-0.11.21/src/audiodecoding/flac/k3bflacdecoder.cpp	2005-03-16 10:23:19.000000000 +0100
@@ -310,7 +310,7 @@
   if( d->comments != 0 ) {
     if( info == i18n("Vendor") )
 #ifdef FLAC_NEWER_THAN_1_1_1
-      return QString::fromUtf8(d->comments->get_vendor_string());
+      return QString::fromUtf8( (const char *) d->comments->get_vendor_string() );
 #else
       return QString::fromUtf8(d->comments->get_vendor_string().get_field());
 #endif
